### ISSUE TITLE
limited number of forecast hours

### ISSUE THREAD

**Issue by opeeters** [irceline/aq-mobile-be#114]
Every now and again the number of forecast hours in the forecast graph is less than it should be. See e.g.:
![screenshot_20190212-104942](https://user-images.githubusercontent.com/323719/52627370-76e22080-2eb5-11e9-86b0-f4ec2bafc9c1.png)

Only 4 forecast hours are given. This is given in the trend.php at the time:
```
"trend": {
  "o3": [
["2019-02-12 04:00:00+01", 1.02239],
["2019-02-12 05:00:00+01", 1.0082],
["2019-02-12 06:00:00+01", 0.854839],
["2019-02-12 07:00:00+01", 0.95092],
["2019-02-12 08:00:00+01", 0.936709],
["2019-02-12 09:00:00+01", 1.04938],
["2019-02-12 10:00:00+01", 1.29861],
["2019-02-12 11:00:00+01", 1.52414],
["2019-02-12 12:00:00+01", 1.86897],
["2019-02-12 13:00:00+01", 2.2],
["2019-02-12 14:00:00+01", 2.35862],
["2019-02-12 15:00:00+01", 2.41379],
["2019-02-12 16:00:00+01", 2.48966],
["2019-02-12 17:00:00+01", 2.44138],
["2019-02-12 18:00:00+01", 2.26207],
["2019-02-12 19:00:00+01", 2.12414],
["2019-02-12 20:00:00+01", 2.07586],
["2019-02-12 21:00:00+01", 2.09655],
["2019-02-12 22:00:00+01", 2.08276]
  ],
  "pm10": [
["2019-02-12 04:00:00+01", 1.03686],
["2019-02-12 05:00:00+01", 1.03184],
["2019-02-12 06:00:00+01", 1.03011],
["2019-02-12 07:00:00+01", 1.02577],
["2019-02-12 08:00:00+01", 1.02171],
["2019-02-12 09:00:00+01", 1.01883],
["2019-02-12 10:00:00+01", 1.01789],
["2019-02-12 11:00:00+01", 1.01572],
["2019-02-12 12:00:00+01", 1.03062],
["2019-02-12 13:00:00+01", 1.0453],
["2019-02-12 14:00:00+01", 1.06144],
["2019-02-12 15:00:00+01", 1.07778],
["2019-02-12 16:00:00+01", 1.09495],
["2019-02-12 17:00:00+01", 1.11398],
["2019-02-12 18:00:00+01", 1.13612],
["2019-02-12 19:00:00+01", 1.16053],
["2019-02-12 20:00:00+01", 1.18473],
["2019-02-12 21:00:00+01", 1.2079],
["2019-02-12 22:00:00+01", 1.23107]
  ],
  "pm25": [
["2019-02-12 04:00:00+01", 1.04771],
["2019-02-12 05:00:00+01", 1.04198],
["2019-02-12 06:00:00+01", 1.03997],
["2019-02-12 07:00:00+01", 1.03511],
["2019-02-12 08:00:00+01", 1.03073],
["2019-02-12 09:00:00+01", 1.0285],
["2019-02-12 10:00:00+01", 1.02761],
["2019-02-12 11:00:00+01", 1.02464],
["2019-02-12 12:00:00+01", 1.0462],
["2019-02-12 13:00:00+01", 1.06621],
["2019-02-12 14:00:00+01", 1.08747],
["2019-02-12 15:00:00+01", 1.10902],
["2019-02-12 16:00:00+01", 1.13274],
["2019-02-12 17:00:00+01", 1.15953],
["2019-02-12 18:00:00+01", 1.19033],
["2019-02-12 19:00:00+01", 1.22421],
["2019-02-12 20:00:00+01", 1.25778],
["2019-02-12 21:00:00+01", 1.28827],
["2019-02-12 22:00:00+01", 1.31629]  
  ]
}
```
There should be forecasts until 22h00..

**Comment by opeeters** [comment]
This is probably cache related. Clearing the cache via the app settings fixes the issue.

**Comment by janschulte** [comment]
Yes was cache related. Is fixed with: 8d490463a37e689438a82398b4b4005be631ec71
